README.md

```diff
@@ -19,17 +19,76 @@ To install use pip:
 Usage
 -----
 
+### 1. Plot both bandstucture and density of states (DOS) side by side
```

Comment:
I'm pretty sure that this should be "band structure", compare: https://en.wikipedia.org/wiki/Electronic_band_structure
```suggestion
### 1. Plot both the band structure and the density of states (DOS) side by side
```